dist/api_err.py

```diff
@@ -51,6 +51,11 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         'recovery must be run to continue', '''
         This error is generated when wiredtiger_open is configured
         to return an error if recovery is required to use the database.'''),
+    Error('WT_CACHE_FULL', -31807,
+        'operation would overflow cache', '''
+        This error is generated when wiredtiger_open is configured
+        to run in-memory, and an insert or update operation requires more
+        than the configured cache size to complete.'''),
```

Comment:
@agorrod, should we avoid the word "cache" as much as possible, because we have so many "cache" statistics and information already? Here, we could do `WT_INMEMORY_FULL: operation would overflow in-memory configuration`.